GoogleUtilities/Example/GoogleUtilities.xcodeproj/project.pbxproj

```diff
@@ -829,9 +892,14 @@
 					"$(DEVELOPER_FRAMEWORKS_DIR)",
 				);
 				GCC_PRECOMPILE_PREFIX_HEADER = NO;
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"GUL_UNSWIZZLING_ENABLED=1",
```

Comment:
Adding "$(inherited)" here might be a way to eliminate the need for the post_install Podfile step

Reply:
Refactored the preprocessor flags to only be defined once at the project level and to use $(inherited) everywhere else in the project.

The post_install podfile step is to pass the preprocessor flags when the GoogleUtilities library is built. AFAICT, there's no way to pass those through the GoogleUtilities project definition which is why the post_install podfile step is needed!

Reply:
Recording our offline conversation - These flags should only be used to build the library when running the tests. 3P consumers shouldn't use them, and hence they're being added to the test project's podfile and not the GoogleUtilitites podspec.